tests/test_authz_collector.py

```diff
@@ -18,6 +18,9 @@ def test__init__(self) -> None:
         assert self.azc.resource_name == ""
         assert self.azc.guest_permissions == {}
 
+    def test__repr__(self) -> None:
+        assert str(self.azc) == "<AuthzCollector resource_name=>"
```

Comment:
You should use `repr()` instead of `str()` - I think, I wrote a comment about that before 🤔